app/controllers/admin/settings/mandatories_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Admin
+  module Settings
+    class MandatoriesController < Admin::ApplicationController
```

Comment:
I think it reads awkwardly too but have you see the table names for settings models? `settings_authentications` and `settings_mascots` definitely doesn't quite sound like what is inside. But when I started the work I asked people if they would prefer `x_settings` or `settings_x` and the preference was for the latter because it groups nicely. 

Anyway, at the risk of some redundancy I can rename this controller back to `MandatorySettingsController` it was the first name I had.